ddtrace/ext/ci.py

```diff
@@ -102,8 +102,9 @@ def tags(env=None, cwd=None):
         git_info[WORKSPACE_PATH] = git.extract_workspace_path(cwd=cwd)
     except git.GitNotFoundError:
         log.error("Git executable not found, cannot extract git metadata.")
-    except ValueError:
-        log.error("Error extracting git metadata, received non-zero return code.", exc_info=True)
+    except ValueError as e:
+        debug_mode = asbool(os.getenv("DD_TRACE_DEBUG", default=False))
```

Comment:
```suggestion
        debug_mode = log.isEnabledFor(logging.DEBUG)
```

You'll need to add `import logging` above.

This will help catch it if someone configured debug logging not via `DD_TRACE_DEBUG`.

(It might also be faster than asbool + os.getenv, but I am not super worried about that in this code path.

Reply:
Thanks, I've updated this. There are a couple places in the code were we still check for `DD_TRACE_DEBUG` directly, though, in case you want to take a look (you can grep for `os.environ.get("DD_TRACE_DEBUG"` and for `os.getenv("DD_TRACE_DEBUG"`)